File: eureka/ios/Models/ServicesDef.swift
//
//  ServicesDef.swift
//  LifePlus
//
//  Created by work on 11/09/20.
//

import Foundation

public class ServicesDef {

  internal static let SERVICE_UUID = UUID(uuidString: "4C505732-5F43-5553-544F-4D5F53525600")

  private var CustomService: [UUID: Any] = [:]

  internal func getCustomService() -> [UUID: Any] {
    return CustomService;
  }

  internal func setCustomService(customService: [UUID: Any]) {
    CustomService = customService;
  }
}

File: eureka/android/library/src/main/java/com/lifeplus/Pojo/ServicesDef.java
package com.lifeplus.Pojo;

import java.util.HashMap;
import java.util.UUID;

public class ServicesDef {

    public static final UUID SERVICE_UUID = UUID.fromString("4C505732-5F43-5553-544F-4D5F53525600");

    private  HashMap<UUID,Object> CustomService = new HashMap<>();

    public HashMap<UUID, Object> getCustomService() {
        return CustomService;
    }

    public void setCustomService(HashMap<UUID, Object> customService) {
        CustomService = customService;
    }
}
